packages/shared/src/components/FeedItemComponent.tsx

```diff
@@ -138,7 +139,10 @@ export default function FeedItemComponent({
   );
 
   switch (item.type) {
-    case 'post':
+    case 'post': {
+      const PostTag = useList
+        ? PostList
+        : PostTypeToTag[item.post.type] ?? PostCard;
```

Comment:
I don't think this is right.
if the user has list, we force postShare still right?

What happens when you set it to list now and open a squad, (assumption is you don't see anything, but maybe I'm wrong)

Reply:
Ah perfect! 🙏